diff -urN deps.orig/npm/bin/npm-cli.js deps/npm/bin/npm-cli.js
--- deps.orig/npm/bin/npm-cli.js	2014-08-19 11:19:52.000000000 -0400
+++ deps/npm/bin/npm-cli.js	2014-08-28 21:10:20.292000000 -0400
@@ -1,6 +1,10 @@
 #!/usr/bin/env node
 ;(function () { // wrapper in case we're in module_context mode
 
+// NodeSource modification to make npm pick up python2.6 on
+// RHEL5 / CentOS 5
+process.env.PYTHON = '/usr/bin/python2.6';
+
 // windows: running "npm blah" in this folder will invoke WSH, not node.
 if (typeof WScript !== "undefined") {
   WScript.echo("npm does not work when run\n"
